ydb/core/grpc_services/rpc_read_rows.cpp

```diff
@@ -592,7 +592,7 @@ class TReadRowsRPC : public TActorBootstrapped<TReadRowsRPC> {
             }
             case NScheme::NTypeIds::Decimal: {
                 return NYdb::TTypeBuilder().Decimal(NYdb::TDecimalType(
-                        typeInfo.GetDecimalType().GetPrecision(), 
+                        typeInfo.GetDecimalType().GetPrecision(),
```

Comment:
```suggestion
                        typeInfo.GetDecimalType().GetPrecision(), 
```

Да, пробел лишний. Но предлагаю не добавлять неревантных изменений.